root/static/scripts/common/components/EntityLink.js

```diff
@@ -448,6 +459,15 @@ component EntityLink(
     );
   }
 
+  if (creditedAs !== null) {
+    parts.push(
+      ' ',
+      bracketed(
+        <span>{texp.l('as “{name}”', {name: creditedAs})}</span>,
```

Comment:
We have an existing `as “{credit}”` string (I think only used in root/components/InstrumentRelTypes.js currently). Would it make sense to reuse that instead?

Reply:
Do you mean that there's some way to share the string between the two files so the translations can be shared, or something else? (Is the larger i18n/l10n picture documented somewhere? I'm still a bit fuzzy on how it all fits together.)

Reply:
Yes, if you use exactly the same string then translators only need to translate it once. This seems like a good case to reuse, since it's exactly the same concept :)

Reply:
Got it, thanks! I've updated the string to match the existing one.

(I wonder how hard it'd be to check for other strings that differ only in placeholder names...)